apps/server/src/shared/aggregators.test.ts

```diff
@@ -1,211 +1,203 @@
-    supports_identification: true,
-    supports_verification: true,
-    supports_history: true
-  },
   sophtron: {
-    id: 'sophtron',
+    id: "sophtron",
     supports_aggregation: true,
     supports_oauth: true,
     supports_identification: true,
     supports_verification: true,
-    supports_history: true
-  }
-}
+    supports_history: true,
+  },
+};
 
-const allAggregators = [Aggregators.MX, Aggregators.SOPHTRON]
+const allAggregators = [Aggregators.SOPHTRON];
 
-const filterOutAggregator = (aggregator: Aggregators) =>
-  allAggregators.filter((currentAggregator) => currentAggregator !== aggregator)
+const filterOutAggregator = (aggregator: Aggregators): Aggregators[] =>
+  allAggregators.filter(
+    (currentAggregator) => currentAggregator !== aggregator,
+  );
 
 const generateAggregatorTests = (aggregator: Aggregators) =>
   // eslint-disable-next-line @typescript-eslint/no-confusing-void-expression
   describe(`getAvailableAggregators tests for aggregator: ${aggregator}`, () => {
     it(`doesnt return ${aggregator} if it's not in the supported aggregators`, () => {
-      const aggregatorsWithoutCurrentAggregator = filterOutAggregator(aggregator)
+      const aggregatorsWithoutCurrentAggregator =
+        filterOutAggregator(aggregator);
 
       expect(
         getAvailableAggregators({
           institution: institutionAggregatorsSupportEverything,
           jobType: MappedJobTypes.AGGREGATE,
           shouldRequireFullSupport: false,
-          supportedAggregators: aggregatorsWithoutCurrentAggregator
-        })
-      ).toEqual(aggregatorsWithoutCurrentAggregator)
-    })
+          supportedAggregators:
+            aggregatorsWithoutCurrentAggregator as Aggregator[],
+        }),
+      ).toEqual(aggregatorsWithoutCurrentAggregator);
+    });
 
     it(`doesnt return ${aggregator} if it's not in the institution`, () => {
       expect(
         getAvailableAggregators({
           institution: {
             ...institutionAggregatorsSupportEverything,
-            [aggregator]: {}
+            [aggregator]: {},
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
           } as any,
           jobType: MappedJobTypes.AGGREGATE,
           shouldRequireFullSupport: false,
-          supportedAggregators: allAggregators
-        })
-      ).toEqual(filterOutAggregator(aggregator))
-    })
+          supportedAggregators: allAggregators as Aggregator[],
+        }),
+      ).toEqual(filterOutAggregator(aggregator));
+    });
 
     it(`doesnt return ${aggregator} if job type is all and supports_verification is falsy`, () => {
       expect(
         getAvailableAggregators({
           institution: {
             ...institutionAggregatorsSupportEverything,
             [aggregator]: {
-              ...institutionAggregatorsSupportEverything.mx,
-              supports_verification: false
-            }
+              ...institutionAggregatorsSupportEverything.sophtron,
```

Comment:
This really should have been switched to use the testAdapter. As is you'll just have to switch it to testAdapter when sophtron gets removed.

Reply:
Done